docs/plugins.md

```diff
@@ -17,6 +17,8 @@ git clone <git url>
 dokku plugins-install
 ```
 
+> todo: add a command to dokku to install a plugin, given a git repository `dokku plugin:install <git url>`?
```

Comment:
We probably don't want this line here. However it's worth opening an enhancement issue to request this feature or a PR implementing it.